src/main/java/com/dougnoel/sentinel/files/CsvFile.java

```diff
@@ -0,0 +1,334 @@
+package com.dougnoel.sentinel.files;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import com.dougnoel.sentinel.exceptions.FileException;
+import com.dougnoel.sentinel.strings.SentinelStringUtils;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.lang3.StringUtils;
+
+public class CsvFile extends TestFile{
```

Comment:
I'd suggest adding functionality to switch between worksheets (tabs)

Reply:
Csv files don't have worksheets. That's an excel thing that will have to be implemented later.

Reply:
Right! Sorry, got ahead of myself there.